PaperSummaries/Section_6-Survey_Notes.md

```diff
@@ -0,0 +1,35 @@
+# A Survey on Multi-view Learning

```

Comment:
Remove this file from pull request? Upload to google drive if you don't want to lose it